taskcluster/mozillavpn_taskgraph/transforms/add_version.py

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os.path
+
+from taskgraph.transforms.base import TransformSequence
+
+transforms = TransformSequence()
+
+# Append the VPN client version to the filenames of the release-artifacts.
+@transforms.add
+def add_version(config, tasks):
+    # Get the version number from version.txt
+    cwd = os.path.dirname(os.path.realpath(__file__))
+    with open(os.path.join(cwd, '..', '..', '..', 'version.txt'), 'r') as fp:
+        version = fp.readline().strip()
+    
+    # If this is a beetmover promotion job, change the version to "beta" instead
+    if config.kind == "beetmover-promote":
+        version = "beta"
```

Comment:
Does `apt upgrade` still work when the version doesn't change? (Do we care if it doesn't?)

Reply:
ah, right, sure, that's fine